test/SecTester.Bus.Tests/Dispatchers/DefaultMessageSerializerTests.cs

```diff
@@ -65,4 +73,37 @@ public void Deserialize_GenericReturnType_ReturnsObject(string input)
       Foo = "bar"
     });
   }
+  
+  [Theory]
+  [MemberData(nameof(EnumValues))]
+  public void Serialize_GivenEnumValue_ReturnString(Type type, object value, string valueString)
```

Comment:
The `type` parameter is not used (see warnings)

```suggestion
  public void Serialize_GivenEnumValue_ReturnString(object value, string valueString)
```